Meta/ladybird.sh

```diff
@@ -142,6 +142,15 @@ run_gdb() {
     local GDB_ARGS=()
     local PASS_ARG_TO_GDB=""
     local LAGOM_EXECUTABLE=""
+    local GDB=gdb
+    if ! command -v "$GDB" > /dev/null 2>&1; then
+      echo "gdb not found, trying lldb"
```

Comment:
seems like this would get pretty verbose if it prints every time we launch the debugger

Reply:
true, removed.